### ISSUE TITLE
Piwik docker in Rancher

### ISSUE THREAD

**Issue by larrybek** [piwik/piwik#10650]
Hi there,

I hope you can help me with. Im running piwik docker in Rancher. 

For some reason all my visits that are called by tracker code in website got ip of 10.40.0.0 which is ip of Rancher server.

Can you help me to make that Piwik get the right ips for visits?

**Comment by mattab** [comment]
Hello, do you use our official docker Piwik image project?

**Comment by larrybek** [comment]
Yes Im using original piwik docker

**Comment by mattab** [comment]
Did you configure it any differently than documented @ https://github.com/piwik/docker-piwik 
I'm wondering why you have this issue while others don't. maybe it could be our documentation that is missing something, or your setup that is a bit different somehow?

**Comment by larrybek** [comment]
Strange thing I created a docker based on piwik docker added nginx as you described in your doc for using Piwik with nginx. Only I need that nginx and piwik be in one file as Rancher is not allowing usage of volume from. But the case that Im not getting out of local network and getting ip of Rancher internal network which is 10.42.0.0. Also are you familiar with Rancher so this will give you clear imagination why its not working or so. Piwik is working out of Rancher without issue I tested it. But issue is only if I run Piwik and Nginx under Rancher.
